src/models/Trigger.ts

```diff
@@ -0,0 +1,10 @@
+import { Model, Schema, model } from "mongoose";
+
+import { ITrigger } from "types/mongodb";
+
+const TriggerSchema = new Schema({
```

Comment:
Triggers should probably include the name of the trigger to separate them from the other ones
(so you don't opt out of all triggers all at once)